src/daemon.rs

```diff
@@ -192,6 +213,12 @@ impl Checker for Daemon {
 
         self.timeout_battery = time::Duration::from_millis(self.settings.delay_battery);
         self.timeout = time::Duration::from_millis(self.settings.delay);
+
+        if let Some(name) = &self.settings.csv_file {
+            if !Path::new(name).exists() {
+                File::create(name).expect("CSV file could not be created.");
```

Comment:
Maybe don't expect here we can easily return and error and continue.

Reply:
![image](https://user-images.githubusercontent.com/35516367/198162739-c66c353d-3f6c-434d-94d9-71ccaa3d287d.png)

Reply:
just pushed this fix

Reply:
also can u add a simple `if confition > 100 then condition =100`? cuz my battery condition is currently at 102% :joy:

Reply:
You guys protect your batteries from degrading? I like to store mine in the fridge.
![image](https://user-images.githubusercontent.com/57497866/198163394-3afaed1e-86db-4774-a187-5ae14368a50e.png)

Reply:
lmao
(also u might wanna check ur batterys physical condition to make sure it's not a spicy pillow)

Reply:
> lmao (also u might wanna check ur batterys physical condition to make sure it's not a spicy pillow)

Well, the battery is new-ish (around a year old). It holds a charge okay, but it's not near spicy pillow levels yet.

Reply:
Oh, we gotta fix the error where turbo does not work on amd

Reply:
Mine just gives the system os error haha

Reply:
Fixed